app/controllers/orders/new.js

```diff
@@ -6,6 +6,7 @@ export default Controller.extend({
       try {
         this.set('isLoading', true);
         let order = data;
+        await order.user.get('save');
```

Comment:
`this.get('authManager.currentUser').save()`
and this save should be conditional --> only if the value for the name  was previously null, and now there is a name provided or edited, we make this extra call.

Reply:
@CosmicCoder96 Okay I will test this out

Reply:
@CosmicCoder96 Okay perfect, it works. Pushing the changes soon